tools/protojsonschema_with_aspects/BUILD

```diff
@@ -0,0 +1,13 @@
+load("//tools/protojsonschema_with_aspects:protojsonschema.bzl", "protojsonschema_rule")
```

Comment:
rather dont include _any_ use/rules/aspects etc in this rule - lets just get the binary working